extensions/memory-operands-2/README.md

```diff
@@ -49,6 +49,10 @@ The behavior when `REX.Q` is set while in `quad` addressing mode when _any_ of t
   - The instruction has an `iS` and a `d8`
   - The instruction has a `dP` and an `i8`
 
+## Exceptions
+
+Using this extension with any instruction that would normally access memory, including but not limited to `store`, `load`, `push`, and `pop`, is _unspecified_ behavior.
```

Comment:
This is specified (with less clear wording IMO) right underneath this. Can we pick one or the other and not both?

Reply:
Looks like only MO2 specifies it twice. Fixed